### DISCUSSION TITLE
Identifizieren von einzelnem Feature führt zu fehlender Werkzeugleiste

### DISCUSSION THREAD

**Discussion by k-placer** [e-netze/webgis-community#68] [Q&A]
Bei mehreren selektierten Flächen funktioniert die gelbe Selektion einzelner Flächen durch Antippen einer Listeneintrags. Die Werkzeuge Objekte selektieren, Nachbarschaft, Tabelle und "Ergebnisse entfernen" sind vorhanden:
![image](https://user-images.githubusercontent.com/97947925/236151382-f3ab1348-3bf7-4943-bbf7-9761802ff413.png)

Wenn ich in der mobilen Darstellung ein Feature abfrage, kann ich die Fläche nicht gelb selektieren und die Werkzeuge Objekte selektieren, Nachbarschaft, Tabelle und Ergebnisse entfernen fehlen:
![image](https://user-images.githubusercontent.com/97947925/236150828-26a5092f-09b9-4c4d-ab9e-76aff6375378.png)

 ### Nachtrag zur Desktop Variante
Hier hat man bekanntlich 2 Darstellungen

- Fenster links, unterhalb von Darstellung hat auch bei einem Feature die Werkzeuge
- Fenster unten, im linken unteren Kartenfenster hat keine Tools

![image](https://user-images.githubusercontent.com/97947925/236178094-02192f12-f6c2-49ac-a272-501fc8fc6478.png)

Kann man die mobile Variante so ändern, dass die Werkzeuge auch bei einem einzigen Feature angeboten werden?

**Comment by gis-eni** [comment]
Stimmt, die Usability ist hier nicht ideal. Darum gibt das im nächsten Build  5.23.2001 dazu ein paar Verbesserungen:

1) Es macht keinen Sinn, dass weniger Werkzeuge angeboten werden, wenn nur ein Grundstück angezeigt wird. Daher schaut das in Zukunft so aus:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/2cdcde20-5d4c-4c20-a00e-bc6f3bf9f309)

Es werden im Prinzip bei einem Ergebnis nur die Pfeile zum Navigieren zwischen einzelnen Ergebnissen ausgeblendet:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/6dc22ec2-7f02-427a-97c2-edb78943aaf0)

2) Mobile Anwendungen:

In der Werkzeugliste für ein einzelnen Objekt werden nicht alle Werkzeuge angeboten. Das betrifft Werkzeuge die alle Objekte betreffen und nicht nur das einzelne Objekte, zB ``Nachbarschaft/Puffern`` berechnen. Auch wenn nur ein Objekt ausgewählt ist, müsste der Anwender, um zu ``Puffern``, den Einzel-Ergebnis Dialog schließen und zur "Liste" der Ergebnisse zurück gehen. Da kann er dann beispielsweise auch``Puffern``.

Das ist allerdings ein bisserl umständlich. Um hier die Usability zu verbessern, gibt es in der Mobilen Variante ab jetzt einen zusätzlichen Button ``Alle Ergebnisse``, mit dem man zur Liste zurück kommt (Weg kommt dafür der ``Tabelle`` Button, damit ein bisserl Platz gespart wird. Tabelle ist Mobile eh eher schwierig und wenn es wer braucht, steht der Button immer noch bei den Werkezeugen in der Liste zur Verfügung):

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/2432159a-6980-46c0-9418-569439e599be)

Von hier aus kann der Anwender kann einfach das entsprechende Tool verwenden oder einfach ein anderes Objekt für die Detailansicht auswählen.
